sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -1042,7 +1042,7 @@ public Mono<RecognizeLinkedEntitiesResultCollection> recognizeLinkedEntitiesBatc
      * {@link RecognizeLinkedEntitiesResultCollection}.
      *
      * @throws NullPointerException if {@code documents} is null.
-     * @throws IllegalStateException if {@link TextAnalyticsRequestOptions#isServiceLogsDisabled()} is true in service
+     * @throws UnsupportedOperationException if {@link TextAnalyticsRequestOptions#isServiceLogsDisabled()} is true in service
```

Comment:
is this not a code change?

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/30850/files#diff-59cdc98fcb42e77bf0ffe2a9ceea1446d3887fb4f179f67d5258e7abfa89f4d3R1367?